src/ontology/edits/oeo-physical.omn

```diff
@@ -1621,28 +1746,24 @@ Class: HeatDemand
         Demand
     
     
-Class: HeatGenerator
-
-    SubClassOf: 
-        Generator
-    
-    
-Class: HeatGeneratorTechnology
+Class: HeatPump
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "A heat generator technology is an technology used to transform energy from some source into heat."@en
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A heat pump is a heater that transforms low temperature heat to high temperature heat using external energy."@en
     
     SubClassOf: 
-        EnergyGeneratorTechnology
+        Heater
     
     
-Class: HeatPumpHeatGenerator
+Class: Heater
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "A heat pump heat generator is a heat generator that uses a heat pump to generate heat."@en
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A Heater is an energy converting device that converts other forms of energy into useful heat.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/173
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/273"
     
     SubClassOf: 
-        HeatGenerator
+        <http://openenergy-platform.org/ontology/oeo-physical/EnergyConvertingDevice>
```

Comment:
* In line with `Generator` we should add the relationship `uses some EnergyCarrier`.
* We should include _has_physical_output some Heat`.

Reply:
I added _has_physical_output some Heat_.
We don't have a class EnergyCarrier right now. Just EnergyCarrierDisposition and Fuel.
So I could add "uses some fuel" or "uses some ('has disposition' some EnergyCarrierDisposition') to Generator and Heater or directly to EnergyConvertingDevice?

Reply:
Okay, so this branch is currently not up to date with the dev branch?

Reply:
def has no EnergyCarrier either

Reply:
we can add an EnergyCarrier class. Would just be an equivalent class to "PortionOfMatter and 'has disposition' some EnergyCarrierDisposition'"

Reply:
Yes, I know that dev has no class EnergyCarrier as it was renamed to EnergyCarrierDisposition. But the fact that *this* branch still has the EnergyCarrier class means that it is not up to date with the dev branch.

Reply:
I can't find an EnergyCarrier class on this branch

Reply:
There it is:
![grafik](https://user-images.githubusercontent.com/36884905/77898805-05615f00-727c-11ea-9f59-9140b5e8706e.png)

Reply:
fascinating, that's not the case in my file. Are you sure your local branch is up to date?

Reply:
Yes. But it seems to be a problem of Protégé with loading the imports. I create an issue.

Reply:
Okay, issue #293 created. So I resolve this conversation as it is only an artifact.